service-impl/config-service-impl/rest-config-service/src/main/java/uk/gov/gchq/palisade/config/service/impl/ApplicationConfigV1.java

```diff
@@ -20,16 +20,26 @@
 import uk.gov.gchq.palisade.rest.ServiceBinder;
 import uk.gov.gchq.palisade.rest.application.AbstractApplicationConfigV1;
 
+import static java.util.Objects.isNull;
+
 public class ApplicationConfigV1 extends AbstractApplicationConfigV1 {
     private static final Class<?>[] RESOURCES = new Class<?>[]{
             RestConfigServiceV1.class
     };
 
     public ApplicationConfigV1() {
         super(RESOURCES);
+        //check environment variable, if that doesn't work then check system property
+        String bootStrapLocation = System.getenv(RestConfigServiceV1.BOOTSTRAP_CONFIG);
+
+        if (isNull(bootStrapLocation)) {
+            bootStrapLocation = System.getProperty(RestConfigServiceV1.BOOTSTRAP_CONFIG);
+        }
+
```

Comment:
Can we add an error message for if the bootstrap location is still null after checking the env and properties

Reply:
Will be addressed in gh-387